server/src/routes/project.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from "express";
+import { Project } from ".prisma/client";
+
+import { asyncHandler } from "../utils/asyncHandler";
+import { prisma } from "../common";
+import express from "express";
```

Comment:
You have the express import duplicated twice. Please consolidate it into one import.